### PR TITLE
front: fix last e2e test (stdcm test) & remove the enable tag in e2e tests 

### LINKED ISSUE #5510: front: refacto E2E tests: stdcm between two trains
# Description

Refactor `stdcm between two trains` e2e tests according to what's been defined in #5022

# Goal

Make E2E test easier to use and maintain.

# Acceptance criteria

The following e2e have been refectored :
- [ ] stdcm between two trains

# Implementation

- [ ] call the relevant api to add one train
- [ ] call STDCM again

### PR DESCRIPTION
**Pull request by clarani**
closes #5510

### CODE REVIEW

File: front/tests/pages/simulation-conf-page.ts [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    await this.seachByTrigramRocketButton.click();
+    await expect(this.searchByTrigramContainer).toBeVisible();
+    await this.searchByTrigramInput.fill(`${firstTrigram} ${secondTrigram}`);
+    await expect(
+      this.page.getByTestId(`typeandpath-op-${firstTrigram}`) &&
+        this.page.getByTestId(`typeandpath-op-${secondTrigram}`)
+    ).toBeVisible();
+    await this.searchByTrigramSubmitButton.click();
+    await expect(this.pathfindingDoneLabel).toBeVisible();
+  }
```
Comment (Yohh) [inline front/tests/pages/simulation-conf-page.ts:58]: why did you add this function? it is already in `scenario-page-model.ts`

**Review by Yohh — COMMENTED** [review]
nice! thank you for this last refacto!

1 comment

**Review by Uriel-Sautron — APPROVED** [review]
Tested, thank you for your work

File: front/tests/pages/simulation-conf-page.ts (hunk 1 above)
Comment (clarani) [reply front/tests/pages/simulation-conf-page.ts:58]: Yes I know, but at term, scenario-page-model should extend simulation-conf-page, this way we will be able to remove the function in it :)

Comment (Yohh) [reply front/tests/pages/simulation-conf-page.ts:58]: ok, can you add a comment explaining it?

**Review by Yohh — APPROVED** [review]
LGTM :muscle:

**Review by Math-R — APPROVED** [review]
lgtm :)
